Initial changes for network topology graph tab

prashant-singh1 opened the pull request.

prashant-singh1 committed:
Initial changes for network topology graph tab

Signed-off-by: prashant-singh1 <prashant.singh8521@gmail.com>

prashant-singh1 committed:
Added changes show STA info on click on STA icon
also change the color shade of sinewave connecting sta

Signed-off-by: prashant-singh1 <prashant.singh8521@gmail.com>

rdkcmf-jenkins commented:
b'## Copyright scan failure
 **Commit:** 05567a861284b77a03ad7ad636b317509185cd1c
**Report detail:**  https://gist.github.com/rdkcmf-jenkins/6083b632cad70727800f592f02adfdc6'

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 1 file pending identification.
* Protex Server Path: /home/blackduck/github/unified-wifi-mesh/431/rdkcentral/unified-wifi-mesh

* Commit: 05567a861284b77a03ad7ad636b317509185cd1c

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/bf6564bfd5de1cd69f7426faff0e918a)'

prashant-singh1 commented:
> Hi Prashant: Where did the new images in this pull req come from? (That includes icons and images of iPhone, iPad etc.)

I have downloaded the below icons from the links shared.
Ipad:
https://share.google/images/AgpJ98zolbq6XLniq

android:
https://share.google/images/iJhwdKBOsF5OaYDnt

iphone:
https://share.google/images/l8AfTxAoPJ4KRx6Ga

few icons have been shared by @soumyasmunshi , Will check with him on the source

prashant-singh1 committed:
Merge branch 'rdkcentral:main' into fyecli_topo

rdkcmf-jenkins commented:
b'## Copyright scan failure
 **Commit:** 3e05dccad088ebd740be7e90f3613bebf00d8396
**Report detail:**  https://gist.github.com/rdkcmf-jenkins/5f11853685139edb887a2a5df4a315b0'

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 1 file pending identification.
* Protex Server Path: /home/blackduck/github/unified-wifi-mesh/431/rdkcentral/unified-wifi-mesh

* Commit: 3e05dccad088ebd740be7e90f3613bebf00d8396

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/923307022575ffaab7e44868c8a9e440)'

mhughesacn reviewed (changes requested):
Thanks Prashant.  These three images cannot be used because there are associated licenses with conditions.  To be honest, the easiest way to get images of iPhone, iPad and Android phone would be to take your own photographs which would come without any external license obligations.  Otherwise, you need to find a suitable image site - pexels and pixabay licenses were allowed by Comcast Legal in the past, also some of the Creative Commons licenses are good.  But you will have to store in separate directories with more license info.

The images from Android and iPhone/iPad also contain Google or Apple content, but I don't think this is worth pursuing especially if it's a photo of someone's phone ("Fair Use").

Also, where does topology.go come from?  All significant code files should have headers unless autogenerated.

Still interested in the other icons as well.

Thank you!

prashant-singh1 committed:
Initial changes for live rendering of topology graph

Signed-off-by: prashant-singh1 <prashant.singh8521@gmail.com>

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 1 file pending identification.
* Protex Server Path: /home/blackduck/github/unified-wifi-mesh/431/rdkcentral/unified-wifi-mesh

* Commit: 56bc27363ba00aa9c07310c4ad866088e13910aa

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/68b1bc85536545d0554e83b7971997a1)'

prashant-singh1 committed:
updated go.sum for bubbletea@v1.2.4

Signed-off-by: prashant-singh1 <prashant.singh8521@gmail.com>

rdkcmf-jenkins commented:
b'## Copyright scan failure
 **Commit:** fd1be91536097b0f18867c2684bf8449c112977b
**Report detail:**  https://gist.github.com/rdkcmf-jenkins/d2773267e8817f994e22680b8247c14e'

prashant-singh1 committed:
Corrected the parsing loging to parse correct band from vapmode 1,
hardcorded the IEEE value for haultype
added changes to skil clear previous screen if get_network command fetch failed.

Signed-off-by: prashant-singh1 <prashant.singh8521@gmail.com>

prashant-singh1 committed:
Corrected STA postion within the connected ssid circle
added changes to consider the last entries if multiple entries of "MACAddress":   "00:00:00:00:00:00" is present

Signed-off-by: prashant-singh1 <prashant.singh8521@gmail.com>

rdkcmf-jenkins commented:
b'## Copyright scan failure
 **Commit:** f497ff5a87be1b6309078761fbcf0492ecfc27fd
**Report detail:**  https://gist.github.com/rdkcmf-jenkins/8583385b8204b42ce2ea26a3319868d0'

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 1 file pending identification.
* Protex Server Path: /home/blackduck/github/unified-wifi-mesh/431/rdkcentral/unified-wifi-mesh

* Commit: f497ff5a87be1b6309078761fbcf0492ecfc27fd

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/e8675a6b72ecfd305c11706e30328b00)'

prashant-singh1 committed:
Modified the code to arrange the STA position correctly when sta counts increases
Adjusted the popup message box to display STA info

Signed-off-by: prashant-singh1 <prashant.singh8521@gmail.com>

rdkcmf-jenkins commented:
b'## Copyright scan failure
 **Commit:** c7439ab9f64429d98ce56e7bca27bfca5ce989a1
**Report detail:**  https://gist.github.com/rdkcmf-jenkins/a548bd10872ea18c94f498639e490f3f'

prashant-singh1 committed:
Merge branch 'rdkcentral:main' into fyecli_topo

rdkcmf-jenkins commented:
b'## Copyright scan failure
 **Commit:** 280a84a81bd6898d3bdd034c9918f0c905f13480
**Report detail:**  https://gist.github.com/rdkcmf-jenkins/e31357bfa8121adde7608a2b9924c44d'

prashant-singh1 committed:
Fixed SSL_Write failed issue
updated STA place around the node with concentric layout
updated timer to fetch topology data from controller for every 20 sec instead 5 sec
updated changes to fetch vlanId from live data

Signed-off-by: prashant-singh1 <prashant.singh8521@gmail.com>

prashant-singh1 committed:
Merge branch 'rdkcentral:main' into fyecli_topo

prashant-singh1 committed:
Added changes to display MLD address if supported for BSSList and STAList
Added Laptop icon for STA

Signed-off-by: prashant-singh1 <prashant.singh8521@gmail.com>

prashant-singh1 committed:
Added green shadow around the edge of ssid circle if mld is enabled for that
updated the static json file to have MLD address (for testing)

Signed-off-by: prashant-singh1 <prashant.singh8521@gmail.com>

prashant-singh1 committed:
Merge branch 'rdkcentral:main' into fyecli_topo

prashant-singh1 committed:
Added fix for fynecli crash for channel change

Signed-off-by: prashant-singh1 <prashant.singh8521@gmail.com>

prashant-singh1 committed:
Merge branch 'rdkcentral:main' into fyecli_topo

prashant-singh1 committed:
Resolve ssl_read issue with more than 16K data (failed to get network topology in fynecli)

Signed-off-by: prashant-singh1 <prashant.singh8521@gmail.com>

rdkcmf-jenkins commented:
b'## Copyright scan failure
 **Commit:** ccf5704064a94425b5f9d031b219bf518dcdacd1
**Report detail:**  https://gist.github.com/rdkcmf-jenkins/0e722889aa8f6e5588f0c81ee8adf55f'

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 1 file pending identification.
* Protex Server Path: /home/blackduck/github/unified-wifi-mesh/431/rdkcentral/unified-wifi-mesh

* Commit: ccf5704064a94425b5f9d031b219bf518dcdacd1

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/120266edb1547b951965314e8823a5c6)'

prashant-singh1 committed:
Merge branch 'rdkcentral:main' into fyecli_topo

rdkcmf-jenkins commented:
b'## Copyright scan failure
 **Commit:** e47850a0a121ccd006163ab52f875b03f564f8c4
**Report detail:**  https://gist.github.com/rdkcmf-jenkins/af18e1a2c5956b8ad4dff4226a6706e5'

prashant-singh1 committed:
Merge branch 'rdkcentral:main' into fyecli_topo

rdkcmf-jenkins commented:
b'## Copyright scan failure
 **Commit:** e3a6cf9fb5ec2144d9ca2185f6f83eb6ee273e6a
**Report detail:**  https://gist.github.com/rdkcmf-jenkins/d687770194f4382632aba6b79bdfdb18'

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 1 file pending identification.
* Protex Server Path: /home/blackduck/github/unified-wifi-mesh/431/rdkcentral/unified-wifi-mesh

* Commit: e3a6cf9fb5ec2144d9ca2185f6f83eb6ee273e6a

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/aa242afc2fe1c934fe0a413beb23eb88)'

amarnathhullur commented:
This change is part of fynecli which is deprecated now. Thus closing this PR
